requirements.txt

```diff
@@ -0,0 +1,4 @@
+flask
```

Comment:
In your requirements file it would be better to state the needed version as well.